src/IJ_mv/IJMatrix_parcsr.c

```diff
@@ -363,7 +368,7 @@ HYPRE_Int hypre_IJMatrixGetRowCountsParCSR( hypre_IJMatrix *matrix,
    hypre_MPI_Comm_rank(comm,&my_id);
 
 #ifdef HYPRE_USING_OPENMP
-#pragma omp parallel for private(i, row_index) HYPRE_SMP_SCHEDULE
+   #pragma omp parallel for private(i, row_index) HYPRE_SMP_SCHEDULE
```

Comment:
I'm not sure why astyle indents `#pragma` this way (it doesn't do this for other preprocessor commands).  I looked around online, and some people prefer this approach.  Pre-ANSI standards used to require preprocessor commands to start at the beginning of the line, but that's not common practice today.  So, I think this is okay, and I don't think there is a nice way to change the style file to prevent this without having an impact on other preprocessor commands in a way that we probably wouldn't want.